Sunrise/ViewModels/CartViewModel.swift

```diff
@@ -39,31 +39,35 @@ class CartViewModel: BaseViewModel {
 
     // MARK: - Data Source
 
-    func numberOfItemsInSection(section: Int) -> Int {
+    func numberOfRowsInSection(section: Int) -> Int {
         return cart.value?.lineItems?.count ?? 0
     }
 
-    func itemNameAtIndexPath(indexPath: NSIndexPath) -> String {
+    func cartItemNameAtIndexPath(indexPath: NSIndexPath) -> String {
```

Comment:
Maybe I was unclear, but I meant `lineItemNameAtIndexPath`, not `cartItemNameAtIndexPath`. The `cart` prefix is not important since you are in the `CartViewModel`, but the `lineItem` name is important.